pkg/printers/internalversion/describe.go

```diff
@@ -856,6 +856,26 @@ func printISCSIVolumeSource(iscsi *api.ISCSIVolumeSource, w PrefixWriter) {
 		iscsi.TargetPortal, iscsi.IQN, iscsi.Lun, iscsi.ISCSIInterface, iscsi.FSType, iscsi.ReadOnly, iscsi.Portals, iscsi.DiscoveryCHAPAuth, iscsi.SessionCHAPAuth, iscsi.SecretRef, initiator)
 }
 
+func printISCSIPersistentVolumeSource(iscsi *api.ISCSIPersistentVolumeSource, w PrefixWriter) {
+	initiatorName := "<none>"
+	if iscsi.InitiatorName != nil {
+		initiatorName = *iscsi.InitiatorName
+	}
+	w.Write(LEVEL_2, "Type:\tISCSI (an ISCSI Disk resource that is attached to a kubelet's host machine and then exposed to the pod)\n"+
+		"    TargetPortal:\t%v\n"+
+		"    IQN:\t%v\n"+
+		"    Lun:\t%v\n"+
+		"    ISCSIInterface\t%v\n"+
+		"    FSType:\t%v\n"+
+		"    ReadOnly:\t%v\n"+
+		"    Portals:\t%v\n"+
+		"    DiscoveryCHAPAuth:\t%v\n"+
+		"    SessionCHAPAuth:\t%v\n"+
+		"    SecretRef:\t%v\n"+
```

Comment:
@rootfs 
Can you show item name like this? Also namespace first would be good.
"SecretRef:\tNamespace:%v, Name:%v\n"+"

Currently, SecretRef is shown like this but adding item name is easy to read.
SecretRef:          &{chap-secret default}

Reply:
Actually, adding item name to right side wasn't good idea. I withdraw above comment.
```
    Type:               ISCSI (an ISCSI Disk resource that is attached to a kubelet's host machine and then exposed to the pod)
    TargetPortal:       x.x.x.x:3260
    IQN:                iqn.2017-05.com.example:rhel7
    Lun:                0
    ISCSIInterface      default
    FSType:             ext4
    ReadOnly:           true
    Portals:            [x.x.x.x:3260]
    DiscoveryCHAPAuth:  false
    SessionCHAPAuth:    true
    SecretRef:          Namespace:default, Name:chap-secret
    InitiatorName:      <none>
```